BookingQueueSubscriber/BookingQueueSubscriber.Services/MessageHandlers/HearingParticipantsUpdatedHandler.cs

```diff
@@ -0,0 +1,46 @@
+using System.Linq;
+using System.Threading.Tasks;
+using BookingQueueSubscriber.Services.IntegrationEvents;
+using BookingQueueSubscriber.Services.Mappers;
+using BookingQueueSubscriber.Services.MessageHandlers.Core;
+using BookingQueueSubscriber.Services.VideoApi;
+using Microsoft.Extensions.Logging;
+using VideoApi.Contract.Requests;
+
+namespace BookingQueueSubscriber.Services.MessageHandlers
+{
+    public class HearingParticipantsUpdatedHandler : IMessageHandler<HearingParticipantsUpdatedIntegrationEvent>
+    {
+        private readonly IVideoApiService _videoApiService;
+        private readonly ILogger<HearingParticipantsUpdatedHandler> _logger;
```

Comment:
the logger doesn't appeared to be used, I would suggest either removing it or using it.

Reply:
Good spot. Will remove as there is a log message in the video API service which is called by this handler.